envcontext.test.js

```diff
@@ -41,10 +39,3 @@ test("sdkman tests", () => {
     expect(isSdkmanToolAvailable("java", "22.0.1-tem")).toBeTruthy();
   }
 });
-
-test("nvm tests", () => {
-  if (process.env?.SDKMAN_VERSION) {
```

Comment:
Was the use of SDKMAN_VERSION instead of NVM_DIR the issue here?

Reply:
yes, I have fixed this